EXE/subtitles.xml

```diff
@@ -187,43 +190,43 @@
 <Static AudioFile="mod\m04\m04_offer_0090_trent.wav" TextArea="Cinematic">See you there.</Static>
 
 
-<Static AudioFile="echo_m04.utf|0xAAD01AC4" TextArea="Cinematic">This is commander Marcus Creitmire of the Hessian Lions spec ops unit.</Static>
-<Static AudioFile="echo_m04.utf|0x96D304C4" TextArea="Cinematic">Turn your weapon systems off immediately, and follow our lead. If you try to resist, we have the authority to destroy you.</Static>
+<Static AudioFile="echo_m04.utf|0xAAD01AC4" TextArea="Cinematic">This is commander Marcus Creitmire of the Hessian Lions special operations unit.</Static>
+<Static AudioFile="echo_m04.utf|0x96D304C4" TextArea="Cinematic">Power down your weapon systems immediately, and follow our lead. If you try to resist, we have the authority to destroy you.</Static>
 <Static AudioFile="echo_m04.utf|0x9B797FCD" TextArea="Cinematic">Herr Creitmire, you're being misled! Manipulated! Let go of the escort!</Static>
-<Static AudioFile="echo_m04.utf|0x89A49807" TextArea="Cinematic">Words – they take too much time. Torpedoes are faster.</Static>
+<Static AudioFile="echo_m04.utf|0x89A49807" TextArea="Cinematic">Words - they take too much time. Torpedoes are faster.</Static>
 
 <Static AudioFile="mod\m04\m04_reward_0010_hatcher.wav" TextArea="Cinematic">Gentlemen.</Static>
 <Static AudioFile="mod\m04\m04_reward_0020_trent.wav" TextArea="Cinematic">Miss Hatcher.</Static>
 <Static AudioFile="mod\m04\m04_reward_0030_alaric.wav" TextArea="Cinematic">Miss Hatcher, you have such lovely, deep eyes.</Static>
-<Static AudioFile="mod\m04\m04_reward_0040_hatcher.wav" TextArea="Cinematic">Cut this nonsense immediately, you don’t want to irritate me.</Static>
-<Static AudioFile="mod\m04\m04_reward_0045_hatcher.wav" TextArea="Cinematic">Let’s get to our business. You’ve done a fine job.</Static>
-<Static AudioFile="mod\m04\m04_reward_0050_hatcher.wav" TextArea="Cinematic">I won’t bother asking how you managed to escape Rheinland with THIS kind of documents.</Static>
-<Static AudioFile="mod\m04\m04_reward_0055_hatcher.wav" TextArea="Cinematic">I’m a delicate lady, I’m afraid I won’t be able to handle it.</Static>
+<Static AudioFile="mod\m04\m04_reward_0040_hatcher.wav" TextArea="Cinematic">Cut this nonsense immediately, you don't want to irritate me.</Static>
+<Static AudioFile="mod\m04\m04_reward_0045_hatcher.wav" TextArea="Cinematic">Let's get to our business. You've done a fine job.</Static>
+<Static AudioFile="mod\m04\m04_reward_0050_hatcher.wav" TextArea="Cinematic">I will avoid asking how you managed to escape Rheinland with THIS kind of material.</Static>
```

Comment:
Rather, "I won't even bother" or "I wouldn't event bother".

Reply:
It can certainly be played with by the actor, but I personally think that this change helps increase the English illustratability of the snarky sarcasm vibe that I am told the character is generally projecting in this scene.
In my opinion, "I wouldn't/won't event bother" seems to make the Hatcher character 'sound' excessively snooty and superior and dismissive, yet she offers the Player and Alaric additional work in the same dialog, indicating respect by saying "Skilled Freelancers such as Yourselves," so I think Hatcher is trying to be Snarky in an entertaining fashion, yet Thankful, and Respectful for a difficult-seeming task completed sucesfully.
What are we aiming for?
 @MechaCheff @Jeider ?

Reply:
Variant "will avoid" approved.